src/app/modules/shared/components/month-picker/month-picker.component.spec.ts

```diff
@@ -61,8 +61,8 @@ describe('MonthPickerComponent', () => {
 
   it('selectMonth should call selectDate', () => {
     spyOn(component, 'selectDate');
-    component.selectMonth(10);
-    expect(component.selectDate).toHaveBeenCalledWith(10, 2020);
+    component.selectMonth(1);
+    expect(component.selectDate).toHaveBeenCalledWith(1, 2021);
```

Comment:
I have some comments about this. We have a deep dependency on the current year. Maybe the solution done by Guido would be better: https://github.com/ioet/time-tracker-ui/pull/601#discussion_r552101062

Reply:
@josepato87  the implementation made to Guido are failing, other option is get the the current month an year to test

Reply:
another option is to get a dynamic month and year to not use static values for example

  fit('selectMonth should call selectDate', () => {
    spyOn(component, 'selectDate');
    const monthSelect = new Date().getMonth();
    const yearSelect = new Date().getFullYear()
    component.selectMonth(monthSelect);
    expect(component.selectDate).toHaveBeenCalledWith(monthSelect, yearSelect);
  });

Reply:
I think it's a better approach since if the current year stay static, it'll be necessary to change it every year.